test/let_spec.rb

```diff
@@ -0,0 +1,45 @@
+BDDSM.describe 'BDDSM::Describe.let' do # rubocop:disable Metrics/BlockLength
```

Comment:
Let's define some new block length, that we are going to respect

Reply:
Metrics/BlockLength:
  Max: 30
  Exclude:
    - test/**/*.rb
ok?